### PR TITLE
Rename Rules to RuleSet

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #153

### CODE REVIEW

File: src/Rule/Each.php [hunk 1]
```diff
@@ -17,15 +17,15 @@ final class Each extends Rule
...
     use HasValidationErrorMessage;
 
-    private Rules $rules;
+    private RuleSet $ruleSet;
 
     private string $incorrectInputMessage = 'Value should be array or iterable.';
     private string $message = '{error} {value} given.';
 
-    public static function rule(Rules $rules): self
+    public static function rule(RuleSet $rules): self
```
Comment (vjik) [inline src/Rule/Each.php:25]:
```suggestion
    public static function rule(RuleSet $ruleSet): self
```

File: src/Rule/Each.php [hunk 2]
```diff
@@ -17,15 +17,15 @@ final class Each extends Rule
...
     {
         $rule = new self();
-        $rule->rules = $rules;
+        $rule->ruleSet = $rules;
```
Comment (vjik) [inline src/Rule/Each.php:28]:
```suggestion
        $rule->ruleSet = $ruleSet;
```

**Review by vjik — APPROVED** [review]

File: src/Rule/Each.php (hunk 1 above)
Comment (arogachev) [reply src/Rule/Each.php]: :heavy_check_mark:

File: src/Rule/Each.php (hunk 2 above)
Comment (arogachev) [reply src/Rule/Each.php]: :heavy_check_mark:

**Review by samdark — APPROVED** [review]

**Pull request merged**